lib/src/commands/packages/commands/check/commands/licenses.dart

```diff
@@ -84,7 +84,7 @@ class PackagesCheckLicensesCommand extends Command<int> {
 
   @override
   String get description =>
-      'Check packages licenses in a Dart or Flutter project.';
+      "Check packages' licenses in a Dart or Flutter project.";
```

Comment:
👀

Reply:
This change is intentional, I believe we're missing the apostrophe since the licenses are owned by packages. 

> To form the possessive, add apostrophe + s to the noun.